### ISSUE TITLE
The progress bar and timer stops after 2 seconds if I play an audio file from an exFAT drive

### ISSUE THREAD

**Issue by romraptor** [iina/iina#4212]
**System and IINA version:**

- macOS Ventura 13.1 on a M1 Mac Mini
- IINA 1.3.1

**Expected behavior:**
The progress bar should keep moving and the time should be updated accordingly

**Actual behavior:**
Song plays until they end, but the progressbar and timer stops after 2 seconds

<details>
<summary><b>Crash report:</b></summary>

```
```
</details>

<br>

<details>
<summary><b>mpv log:</b></summary>

```
```
</details>

<br>

**Steps to reproduce:**
Open any audio file Wav, FLAC etc (I have tried many)

- [ ] MPV does not have this problem.

**How often does this happen?**
Always
![progress bar and timer](https://user-images.githubusercontent.com/111131657/217080286-3897d7ec-736e-4684-b633-24210de8bb89.png)

**Comment by low-batt** [comment]
This sounds like the problem reported in issue #4051. That regression is fixed by changes in PR #4087. That PR is in review.

I'm going to mark this as a duplicate of #4051 although I'm a bit concerned that the issue title specifically indicates the involvement of an exFAT drive.

**Comment by romraptor** [comment]
The issue sounds very similar. Is there any preview version a regular user like me can test? How do I know when the fix has been included?

**Comment by low-batt** [comment]
Excellent questions.

The next step is for other developers to finish reviewing the proposed fix in PR #4087 and merge the fix into the `develop` branch. At that point the green "Open" label at the top of the PR will change to a purple "Merged". When that happens we usually add the label "progress: fixed/implemented" to the corresponding issue. We leave the issue open as it is still a problem in the released version of IINA. Once a new version of IINA is released the issue will be closed.

At the moment the only way to test the fix is to build IINA using the `fix-4058` branch. Once the fix is merged into the `develop` branch it will be in a [nightly build](https://nightly.iina.io/).  Nightly builds are not signed, so you would have to overcome security objections from macOS to get a nightly build to run.
